content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -867,6 +867,47 @@ is disabled, `.spec.completions` is immutable.
 Use cases for elastic Indexed Jobs include batch workloads which require 
 scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
 
+### Recreation only when Pod has been fully terminated
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+For distributed jobs, it is common to have many replicas running in a job.  
+Some frameworks will require uniqueness for the pod index.  
+In cases like this, it is beneficial to only recreate pods
+based once the pod has been terminated.
+
+When the `JobPodReplacementPolicy`[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is set, you will be able to specify `PodReplacementPolicy` to `Failed`.  
```

Comment:
The PR title says PodRecreationPolicy. Should that be updated?

Reply:
PR title was wrong.  Good catch on that one.  I updated title to say `PodReplacementPolicy`.